boxyhq/jackson
# Support for `login_hint` OIDC parameter

## What does this PR do?



Adds support for `login_hint` param.

## Type of change





## How should this be tested?



- [x]  To test with google, add an OIDC connection in Jackson and add the product in the `allowedProducts` [list](https://github.com/boxyhq/jackson-remix-auth/blob/b63f93c035237f8372b8e675f27a2dc4c5076db6/app/utils.server.ts#L21).  Point the `BOXYHQSSO_ISSUER` to `http://localhost:5225` to use the local jackson service. 
Since the original issue was filed against the remix demo app, testing this change is as simple as adding the query param at  https://github.com/boxyhq/jackson-remix-auth/blob/b63f93c035237f8372b8e675f27a2dc4c5076db6/app/routes/login.tsx#L50.
   ```
        <Form
            method="post"
            action={`/auth/sso?login_hint=aswin@boxyhq.com`}
            className="flex flex-col items-start space-y-4"
            reloadDocument
          >
  ```
With multiple accounts signed into google, this param should skip the account selection page.


- [x] Existing unit/e2e tests

## Checklist:
